.buildkite/common.py

```diff
@@ -89,11 +90,18 @@ def pipeline_to_json(pipeline):
     return json.dumps(pipeline, indent=4, sort_keys=True, ensure_ascii=False)
 
 
-def get_changed_files(branch):
+def get_changed_files():
     """
     Get all files changed since `branch`
     """
-    stdout = subprocess.check_output(["git", "diff", "--name-only", branch])
+    # Files are changed only in context of a PR
+    if os.environ.get("BUILDKITE_PULL_REQUEST") is None:
```

Comment:
```suggestion
    if os.environ.get("BUILDKITE_PULL_REQUEST", "false") == "false":
```

it will only be `None` if the function is executed outside of buildkite altogether. If its executed by a buildkite pipeline, yet not on a PR, then it will be the string `"false"` [[1]]

[1]: https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_PULL_REQUEST